apps/src/code-studio/header.js

```diff
@@ -180,8 +183,8 @@ function shareProject() {
           icon={appOptions.skin.staticAvatar}
           shareUrl={shareUrl}
           isAbusive={dashboard.project.exceedsAbuseThreshold()}
-          isSignedIn={appOptions.isSignedIn}
-          isPublished={dashboard.project.isPublished()}
+          canPublish={canPublish}
+          isPublished={!!dashboard.project.isPublished()}
```

Comment:
This `!!` coercion is also redundant (`isPublished()` will always return a boolean value).